api/src/main/java/org/openmrs/api/UserService.java

```diff
@@ -566,4 +567,10 @@
 	 * @param newPassword the new password
 	 */
 	public void changePasswordUsingActivationKey(String activationKey, String newPassword);
+
+	/**
+	 * @param user the User whose Locale to retrieve
+	 * @return the default Locale of the given user, or the system locale if unspecified
```

Comment:
Would be great to add the @since annotation

Reply:
Yeah, I thought about that too @dkayiwa .  I figured could be added before merging, once the different versions are known.  Since it has to be changed (added to) when up-ported I would assume.

Reply:
By the way, do you intend to use this method outside of the user service?

Reply:
OK, I added a since 2.3.6.  Does it make sense to add the 2.4, 2.5, and 2.6 annotations at this point, or wait until porting up @dkayiwa ?

Reply:
> Does it make sense to add the 2.4, 2.5, and 2.6 annotations at this point, or wait until porting up @dkayiwa ?

Waiting until porting up makes sense!

Reply:
In that case, what do you think about adding a test for this public method?

Reply:
@dkayiwa good point.  I've added a few.  Please have a look.